### ISSUE TITLE
Identify parts of the system where adding Hooks may be useful

### ISSUE THREAD

**Issue by Muttley** [Muttley/foundryvtt-shadowdark#41]
I don't think this is a high priority before initial release, but we should itentify areas of the system that could benefit from triggering custom hooks that 3rd party modules could potentially hook into.

**Comment by bakbakbakbakbak** [comment]
Agreed. Let's review the Hooks setup for like dnd5e, PF2e, and OSE. That should give a baseline at least for ideas.

**Comment by bakbakbakbakbak** [comment]
We could add a hook for when adding talent to make customizing talenta a bit more flexible. We can also use that to ask for input for talents like weapon, air mastery and the spell advantage ones.
